frontend/integration-tests/tests/olm/descriptors.scenario.ts

```diff
@@ -87,6 +87,7 @@ describe('Using OLM descriptor components', () => {
         openAPIV3Schema: {
           properties: {
             spec: {
+              type: 'object',
```

Comment:
Fixed the test CRD to use structural schema in order for validation to be published to `/openapi/v2`.